Test Automation Course/src/inheritance/Masina.java

```diff
@@ -0,0 +1,28 @@
+package inheritance;
+
+public class Masina extends Vehicul {
+
+    private String marca;
+
+    public Masina(String marca, String culoare, int nrRoti) {
+
+        setCuloare(culoare);
```

Comment:
este recomandata utilizarea cuvintului cheie super cind apelezi metode din clasa parinte

Reply:
Am facut modificari. Le vei putea vedea in commitul care il voi face acum.